src/Goerli-DssSpell.sol

```diff
@@ -46,6 +61,24 @@ contract DssSpellAction is DssAction, DssSpellCollateralAction {
         return false;
     }
 
+    address internal immutable DAI = DssExecLib.dai();
+
+    address internal immutable TELEPORT_JOIN = DssExecLib.getChangelogAddress("MCD_JOIN_TELEPORT_FW_A");
```

Comment:
Not too critical here but generally preferred to use the chainlog key name as the capitalized variable name to aid audits further down the code.

Reply:
I raised this above, for this spell it was done to look as close as possible to the 30th August spell.